suite-native/module-authenticity-checks/src/components/DeviceCompromisedBanner.tsx

```diff
@@ -9,13 +9,25 @@ import { useOpenLink } from '@suite-native/link';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { TREZOR_SUPPORT_FW_REVISION_CHECK_FAILED_MOBILE_URL } from '@trezor/urls';
 
-import { deviceCompromisedBannerAtom } from '../DeviceCompromisedBannerAtoms';
+import {
+    DeviceCompromisedBannerVariant,
+    deviceCompromisedBannerAtom,
+} from '../DeviceCompromisedBannerAtoms';
 
-const containerStyle = prepareNativeStyle(utils => ({
-    backgroundColor: utils.colors.backgroundAlertRedSubtleOnElevation0,
-    // MessageSystemBanner critical variant has the same bgColor, so the margin serves to separate them visually
-    marginBottom: utils.spacings.sp1,
-}));
+const containerStyle = prepareNativeStyle<{ bannerVariant: DeviceCompromisedBannerVariant }>(
+    (utils, { bannerVariant }) => {
+        const backgroundColor =
+            bannerVariant === 'other-error'
+                ? utils.colors.backgroundAlertYellowBold
+                : utils.colors.backgroundAlertRedSubtleOnElevation0;
+
+        return {
+            backgroundColor,
+            // MessageSystemBanner critical variant has the same bgColor, so the margin serves to separate them visually
+            marginBottom: utils.spacings.sp1,
+        };
+    },
+);
```

Comment:
nit: small simplification
```suggestion
const containerStyle = prepareNativeStyle<{ bannerVariant: DeviceCompromisedBannerVariant }>(
    (utils, { bannerVariant }) => ({
            backgroundColor: bannerVariant === 'other-error'
                ? utils.colors.backgroundAlertYellowBold
                : utils.colors.backgroundAlertRedSubtleOnElevation,
            // MessageSystemBanner critical variant has the same bgColor, so the margin serves to separate them visually
            marginBottom: utils.spacings.sp1,
        })
);
```